ev-dev-tools/src/ev_cli/ev.py

```diff
@@ -43,7 +43,7 @@
 
 
 def setup_jinja_env():
-    env.globals['timestamp'] = datetime.utcnow
+    env.globals['timestamp'] = datetime.utcnow()
```

Comment:
I don't know why I changed from the function to it's instantanious value
This way all timestamps will be the same for one run of ev-cli, not sure if this is what we want ...

Reply:
This was actually just intended as a hotfix back to the original implementation. With the change from
```python
datetime.utcnow()
```
to
```python
datetime.utcnow
```
we would also need to change the way the year is accessed in [templates/interface-Impl.cpp](https://github.com/EVerest/everest-utils/blob/feature/requirement_lists/ev-dev-tools/src/ev_cli/templates/interface-Impl.cpp.j2#L2) otherwise it just didn't run at all.
However looking at it we should probably remove the year from the license header here as well for consistency with the other autogenerated code.

Reply:
yes we should, the year causes more problems/needed updates than it is doing good.